pints/plot.py

```diff
@@ -8,6 +8,201 @@
 #
 from __future__ import division
 
+def log_likelihood_1d(log_likelihood, parameters, boundaries=None, n_eva=20, chain=None):
+    """
+    Takes a set of parameters as input and creates and returns a plot showing 
+    the log-likelihood function of each parameter with all the other 
+    parameters held at the given set of parameters.
+    
+    Arguments:
+
+    `log_likelihood`
+        A class (or that extends from) :class:`pints.LogLikelihood` of the 
+        problem of interest.
+    `parameters`
+        A set of parameters, which the `log_likelihood` will be evaluated at, 
+        with the same dimension as the `log_likelihood` will accept.
+    `boundaries`
```

Comment:
You need to indicate this is optional (i.e. a chain can be given instead). Also add a check that one or the other is given. (Or combine them into a single argument)